### DISCUSSION TITLE
How would I `source` this plugin?

### DISCUSSION THREAD

**Discussion by mig824** [marlonrichert/zsh-snap#221] [Q&A]
https://github.com/ohmyzsh/ohmyzsh/tree/master/plugins/web-search

**Comment by marlonrichert** [comment]
```zsh
znap source ohmyzsh/ohmyzsh lib/functions.zsh plugins/web-search
```

Documented [here](https://github.com/marlonrichert/zsh-snap/blob/9b8edd89c2b2303ffa253c5116a97d9138e0831d/.zshrc#L45).
